MonoGame.Framework/Graphics/Effect/EffectMaterial.cs

```diff
@@ -5,11 +5,23 @@
 using System;
 namespace Microsoft.Xna.Framework.Graphics
 {
+    /// <summary>
+    /// Contains an effect subclass which is used to load data for an EffectMaterialContent type.
+    /// </summary>
+    /// <remarks>
+    /// For most purposes, this type can be ignored, and treated exactly like a regular effect.
+    /// When an EffectMaterial type is loaded from .xnb format, its parameter values and textures
+    /// are also loaded and automatically set on the effect, in addition to the HLSL shader code.
```

Comment:
We might want to remove the reference to HLSL here. by the time the Effect is being used the onderlying shader will be platform specific , e.g GLSL HLSL, metal etc.